views/default/navigation/breadcrumbs.php

```diff
@@ -25,29 +22,8 @@
 $breadcrumbs = elgg_extract('breadcrumbs', $vars);
 $breadcrumbs = elgg_get_breadcrumbs($breadcrumbs);
 
-if (!is_array($breadcrumbs) || empty($breadcrumbs)) {
-	return;
-}
-
-$attrs['class'] = elgg_extract_class($vars, ['elgg-menu', 'elgg-breadcrumbs']);
-$lis = '';
-
-foreach ($breadcrumbs as $breadcrumb) {
-	// We have to escape text (without double-encoding). Titles in core plugins are HTML escaped
-	// on input, but we can't guarantee that other users of this view and of elgg_push_breadcrumb()
-	// will do so.
-	if (!empty($breadcrumb['link'])) {
-		$crumb = elgg_view('output/url', [
-			'href' => $breadcrumb['link'],
-			'text' => $breadcrumb['title'],
-			'encode_text' => true,
-			'is_trusted' => true,
-		]);
-	} else {
-		$crumb = htmlspecialchars($breadcrumb['title'], ENT_QUOTES, 'UTF-8', false);
-	}
-
-	$lis .= "<li>$crumb</li>";
-}
-
-echo elgg_format_element('ul', $attrs, $lis);
+echo elgg_view_menu('breadcrumbs', [
+	'items' => $breadcrumbs,
```

Comment:
If link/href is empty do these still get output as text, or `a[href=javascript:void()]`?